src/ui/component/claimListDiscover/view.jsx

```diff
@@ -109,6 +111,7 @@ function ClaimListDiscover(props: Props) {
     (personalSort === SEARCH_SORT_CHANNELS && subscribedChannels.length) ||
     (personalSort === SEARCH_SORT_YOU && !!tags.length) ||
     personalSort === SEARCH_SORT_ALL;
+  const hasMatureTags = tags.some(t => MATURE_TAGS.includes(t));
```

Comment:
@seanyesmunt @kauffj Added this in to see if the tag is mature, so the message will be shown accordingly and it shouldn't be too obtrusive anymore!